content-src/asrouter/templates/EOYSnippet/EOYSnippet.schema.json

```diff
@@ -0,0 +1,128 @@
+{
+  "title": "EOYSnippet",
```

Comment:
Missing `monthly_checkbox_label_text`, `locale`, `currency_code` and `test`

Reply:
We could fall back to that but we might want to allow it coming from the server since that's currently in the spec. Should we discuss with giorgos maybe?